pkg/cloud/vsphere/vsphere_config_transformer.go

```diff
@@ -0,0 +1,88 @@
+package vsphere
+
+import (
+	"fmt"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+
+	ccmConfig "github.com/openshift/cluster-cloud-controller-manager-operator/pkg/cloud/vsphere/vsphere_cloud_config"
+)
+
+const (
+	regionLabelValue = "openshift-region"
+	zoneLabelValue   = "openshift-zone"
+)
```

Comment:
I'd like some comments on these, explain what they are, why they are these values, why they can't be configured by a user etc, right now they look like magic consts without any context

Reply:
added, PTAL